scripts/setup.ts

```diff
@@ -27,11 +27,13 @@ has to point to the dev version you want to promote, for example 2.1.0-dev.123`)
     await run('.', `git config --global user.name "prisma-bot"`)
   }
   if (process.env.RELEASE_PROMOTE_DEV) {
-    const versions = await getVersionHashes(process.env.RELEASE_PROMOTE_DEV)
-    // TODO: disable the dry run here
+    // Checkout exactly the commit of the `dev` version that is being promoted/released
```

Comment:
If/when we start to tag all `dev` versions, this whole complicated thing could be replace by looking up the commit of the tag instead of going via Unpkg and `prisma.prismaCommit`, correct?

Reply:
It's also my thinking, and that would be a good thing